src/templates/_footer.hbs

```diff
@@ -1,59 +1,98 @@
 <div class="footer">
     <input id="command"
-        placeholder="enter aircraft command"
-        autocapitalize="off"
-        autocorrect="off"
-        autocomplete="off"
-        spellcheck="false"
-        autofocus />
+           placeholder="enter aircraft command"
+           autocapitalize="off"
+           autocorrect="off"
+           autocomplete="off"
+           spellcheck="false"
+           autofocus />
     {{!-- move to ul li.hlist --}}
-    <ul id="controls">
-        <li class="control toggle-labels" title="Toggle fix and runway labels">
-            <img src="assets/images/icons/labels.png" />
-        </li>
-        <li class="control toggle-sids" title="Toggle SID display">
-            <img src="assets/images/icons/sids.png" />
-        </li>
-        <li class="control toggle-stars" title="Toggle STAR display">
-            <img src="assets/images/icons/stars.png" />
-        </li>
-        <li class="control toggle-restricted-areas" title="Toggle Restricted Areas">
-            <img src="assets/images/icons/restricted.png" />
-        </li>
-        <li class="control toggle-terrain active" title="Toggle Terrain">
-            <img src="assets/images/icons/terrain.png" />
-        </li>
-        <li class="control toggle-video-map active" title="Toggle Video Map">
-            <img src="assets/images/icons/map.png" />
-        </li>
-        <li class="control fast-forwards" title="Set time warp to 2">
-            <img src="assets/images/icons/fast-forwards.png" />
-        </li>
-        <li class="control pause-toggle" title="Pause simulation">
-            <img src="assets/images/icons/pause.png" />
-        </li>
-        <li class="control toggle-speech" title="Enable speech synthesis (experimental)">
-            <img src="assets/images/icons/speech.png" />
-        </li>
-        <li class="control toggle-traffic" title="Adjust traffic rate">
-            <img src="assets/images/icons/traffic.png" />
+
+    <ul>
```

Comment:
this should still use a class name.  directly styling elements has a different specificity that styling classnames (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), and we prefer to style with classnames.